seismostats/utils/__init__.py

```diff
@@ -70,17 +75,34 @@ def wrapper_require(self, *args, **kwargs):
         return decorator_require(_func)
 
 
-def is_nan(value):
+def is_nan(value: float) -> bool:
+    """
+    Check if a value is NaN.
+
+    Args:
+        value:  Value to check.
+
+    Returns:
+        check:  True if the value is NaN, False otherwise.
+
+    Examples:
+        >>> is_nan(float('nan'))
```

Comment:
I think it would be best to import the packages needed in these examples, such that one could paste the code and it works (as long as one has the packages installed). Here, this would be

from seismostats.utils import is_nan

I noticed this in some examples, but didn't comment on all of them..